inc/class-spotim-cron.php

```diff
@@ -109,9 +112,8 @@ public function run_import() {
 
         $this->set_time_limit( 0 );
 
-        $import            = new SpotIM_Import( self::$options, true );
-        $total_posts_count = $import->get_posts_count();

```

Comment:
Is this not required now?

Reply:
`$total_posts_count` was not used so removed.